packages/mui-base/src/SwitchUnstyled/useSwitch.test.tsx

```diff
@@ -9,12 +9,12 @@ import {
   screen,
   simulatePointerDevice,
 } from 'test/utils';
-import { useSwitch, UseSwitchProps, UseSwitchResult } from '@mui/base/SwitchUnstyled';
+import { useSwitch, UseSwitchParameters } from '@mui/base/SwitchUnstyled';
 
 describe('useSwitch', () => {
   const { render } = createRenderer();
-  const invokeUseSwitch = (props: UseSwitchProps): UseSwitchResult => {
-    const ref = React.createRef<UseSwitchResult>();
+  const invokeUseSwitch = (props: UseSwitchParameters) => {
```

Comment:
Let's mark this as breaking change so that we can add it in the changelog for people to be aware of.